cuvs-bench/cpu/Dockerfile

```
@@ -3,7 +3,7 @@
 ARG PYTHON_VER=unset
 ARG RAPIDS_VER=24.12
 
-FROM condaforge/mambaforge:23.3.1-0 AS raft-ann-bench-cpu
+FROM condaforge/mambaforge:23.3.1-0 AS cuvs-bench-cpu
```

This base image is out of date. We need to use miniforge instead of mambaforge, and we should use the latest version.
```suggestion
FROM condaforge/miniforge3:24.7.1-2 AS cuvs-bench-cpu
```

---

It looks like this has stalled in another PR (https://github.com/rapidsai/docker/pull/660/) but that PR was trying to switch to micromamba rather than miniforge3. My suggestion here should be safe, afaik.